cuesubmit/plugins/OpenCue/Setup.py

```diff
@@ -0,0 +1,96 @@
+#  Copyright Contributors to the OpenCue Project
```

Comment:
I think the parent directory should be `cuesubmit/plugins/blender/`.

Reply:
I've set the addon name as **OpenCue** and is detected as such in Blender upon installation, hence the subdirectory title. But I do understand the confusion so moved all the contents inside the subdirectory named blender in https://github.com/AcademySoftwareFoundation/OpenCue/pull/1309/commits/d84505f1d30ffa0354fb1a3cb69e2c30b91ff533.
Added a README in https://github.com/AcademySoftwareFoundation/OpenCue/pull/1309/commits/43f125bc12e8e1e2f402af6a2615669a934b843a in the subdirectory as well so it gives some context to the naming.

Was thinking of setting up the CI/CD pipeline so that it automatically builds the installable OpenCue.zip file along with the source code.